Helper/Cart.php

```diff
@@ -1360,18 +1360,36 @@ function ($item) use ($imageHelper, &$totalAmount, &$diff, $storeId, $currencyCo
                 ////////////////////////////////////
                 // Load item product object
                 ////////////////////////////////////
-                $itemProduct = $item->getProduct();
-                $customizableOptions = $this->getProductCustomizableOptions($itemProduct);
-                $itemSku = trim($item->getSku());
-
-                if ($customizableOptions) {
-                    $itemSku = $this->getProductActualSkuByCustomizableOptions($itemSku, $customizableOptions);
+                //By default this feature switch is enabled.
+                $_product = $item->getProduct();
+                if ($this->deciderHelper->isCustomizableOptionsSupport()) {
+                    try {
+                        $customizableOptions = $this->getProductCustomizableOptions($_product);
+                        if ($customizableOptions) {
+                            $itemSku = $this->getProductActualSkuByCustomizableOptions(trim($item->getSku()), $customizableOptions);
+                            $_product = $this->productRepository->get($itemSku);
+                            $itemReference = $_product->getId();
+                            $itemName = $_product->getName();
+                        }
+                    } catch (\Exception $e) {
+                        $this->bugsnag->registerCallback(function ($report) use ($item) {
+                            $report->setMetaData([
+                                'product_id' => $item->getProductId(),
+                                'product_name' => $item->getName(),
+                                'product_sku' => $item->getSku()
+                            ]);
+                        });
+                        $this->bugsnag->notifyError('Could not retrieve product', $e->getMessage());
+                        $customizableOptions = null;
+                    }
+                }
+                if (empty($customizableOptions)) {
```

Comment:
I know this code works, but could you add `$customizableOptions = null;` before line 1365?

Reply:
Done

Reply:
Could you push your change?